src-v2/Not/Not.Domain/DomainObject.cs

```diff
@@ -1,3 +1,10 @@
 ﻿namespace Not.Domain;
 
-public abstract record DomainObject { }
+public abstract record DomainObject
+{
+    protected string Combine(params object?[] values)
+    {
+        var sections = values.Where(x => x != null);
+        return string.Join(" | ", sections);
+    }
```

Comment:
The logic is copy-paste from `DomainEntity`....

Reply:
Copied this since you have comment to use Combine in ToString(). However, as per convention Start is DomainObject since it is created during runtime and isn't saved. The Combine method was missing from DomainObject.

Reply:
And there isn't a possible solution to reuse code between two objects other than inheritance?

Reply:
First, I thought about a common interface, but then I noticed the DomainExtension class where I can define:
```
public static string Combine(params object?[] values)
{
      var sections = values.Where(x => x != null);
      return string.Join(" | ", sections);
}
```
this would require to add the following using in all DomainEntities:
`using static NTS.Domain.Extensions.DomainExtensions;`